src/resources/users.js

```diff
@@ -51,6 +52,7 @@ class Users extends ResourceBase {
     let identityAddress = await this.identityAddress(address)
     if (identityAddress) {
       let userData = await this.getClaims(identityAddress)
+      userData.identityAddress = identityAddress
       return new UserObject(userData)
```

Comment:
For some reason, it didn't like the spread operator here. 😕

Reply:
@micahalcorn Yep, we don't get to use the spread operator ☹️. I don't remember specifically why off the top of my head, but we can probably enable it with Babel if we want to sometime.